test/uuid-enforcement.test.ts

```diff
@@ -0,0 +1,118 @@
+import { launchAndConnect, testVersion, TEST_TIMEOUT, cleanOpenRuntimes, getRuntimeProcessInfo, getPort } from './multi-runtime-utils';
+import * as assert from 'assert';
+import * as path from 'path';
+import * as fs from 'fs';
+import { conn } from './connect';
+import { Fin, connect, Application } from '../src/main';
+/* tslint:disable:no-invalid-this no-function-expression insecure-random mocha-no-side-effect-code no-empty max-func-body-length */
+let uuidCount = 0;
+let realmCount = 0;
+
+const manifestPath = path.resolve('test/uuid-enforcement-app.json');
+//tslint:disable-next-line
+const manifest = {
+    startup_app: { uuid: '', name: '' },
+    runtime: { version: testVersion, securityRealm: '' }
+};
+let fin: Fin;
+//tslint:disable-next-line
+const updateManifest = (uuid: string, realm = 'uuid-enforcement-realm-' + realmCount++) => {
```

Comment:
:neckbeard: comment, instead of updating a single manifest you could create multiple ones and delete them as you go. Might actually be better as the logic treats manifest paths as a data point.